example/src/main.rs

```diff
@@ -0,0 +1,33 @@
+extern crate civet;
+extern crate conduit;
+extern crate conduit_router;
+
+use std::collections::HashMap;
+use std::io::{self, Cursor};
+use std::sync::mpsc::channel;
+
+use civet::{Config, response, Server};
+use conduit::{Request, Response};
+use conduit_router::{RouteBuilder, params};
```

Comment:
You may want to import the [`RequestParams`](https://github.com/conduit-rust/conduit-router/blob/master/src/lib.rs#L106-L108) trait so you get the `.params()` method on the `Request` itself (a little more ergonomic)